tests/mortar/rotate_mapping_3d.cc

```diff
@@ -0,0 +1,193 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+/**
+ * @brief Mortar: test functions to compute radius, number of subdivisions, and rotate mapping.
+ *
+ */
```

Comment:
Is this up to date? I think this is meant to say that this test these functions in the context of 3D simulations no?